test/inductor/test_torchinductor.py

```diff
@@ -10646,21 +10646,25 @@ def fn2(a: torch.Tensor, b: torch.Tensor) -> torch.Tensor:
             fn2_opt = torch._dynamo.optimize("inductor")(fn2)
 
             a = torch.rand([100, 100], device=GPU_TYPE)
-            b = torch.rand([100], device=GPU_TYPE)
-            torch._dynamo.mark_dynamic(b, 0)
-            inps = [a, b]
-
-            code1 = run_and_get_triton_code(fn1_opt, *inps)
+            b1 = torch.rand([102], device=GPU_TYPE)
+            b2 = torch.rand([100], device=GPU_TYPE)
+            torch._dynamo.mark_dynamic(b1, 0)
+            torch._dynamo.mark_dynamic(b2, 0)
+            inps1 = [a, b1]
+            inps2 = [a, b2]
+
+            # Run fn2 first since it has more restrictive bounds -- to avoid cache hit
             code2 = run_and_get_triton_code(fn2_opt, *inps)
+            code1 = run_and_get_triton_code(fn1_opt, *inps)
```

Comment:
bug here? There's no `inps` list any more

Reply:
this is what happens when i have to run tests on OSS but export from fbcode :/